engine/lib/luajit-ffi-gen/src/impl_item/type_info.rs

```diff
@@ -5,6 +5,7 @@ const RUST_TO_LUA_TYPE_MAP: &[(&str, &str)] = &[
     ("DVec2", "Vec2d"),
     ("Vec2", "Vec2f"),
     ("Vec3", "Vec3f"),
+    ("Vec4", "Vec4f"),
```

Comment:
Can we pre-emptively add DVec3/DVec4 and IVec3 here too?

Reply:
Done